RFCs/0009-Editors-API-RevisionUpdate.md

```diff
@@ -0,0 +1,350 @@
+# 0009 Editors API Revision Update
+
+## Summary
+This note outlines the proposed changes for the JSON protocol between an Editor/IDE
+and the test platform.
+
+## Overview of changes
+
```

Comment:
Add a motivation section elaborating why this v2 is required:
* Size of payload (verbosity)
* Improvements to versioning and ability to handle future breaking changes

Summary of changes:
* Negotiation protocol: Editor and Test Runner, Test Runner and Test Host
* New data structures in v2
* Changes to existing data structures in v2